cpp/config/suites/c/code-review

```diff
@@ -0,0 +1,31 @@
++ semmlecode-cpp-queries/Likely Bugs/Conversion/LossyPointerCast.ql: /Critical/Critical_Correctness/Dangerous Conversions
```

Comment:
These paths aren't correct any more, right?

Reply:
They should be correct in an odasa zip file because the queries are moved to `semmle-cpp-queries` there.

Reply:
OK.